lib/teiserver/matchmaking/queue_server.ex

```diff
@@ -604,7 +621,36 @@ defmodule Teiserver.Matchmaking.QueueServer do
   """
   @spec get_stats(id()) :: {:ok, stats()} | {:error, :not_found}
   def get_stats(queue_id) do
-    via_tuple = QueueRegistry.via_tuple(queue_id)
+    via_tuple = via_tuple(queue_id)
```

Comment:
why the short timeout btw? If you deviate from the default, you should at the very least put a comment explaining why.